### DISCUSSION TITLE
How to build a project

### DISCUSSION THREAD

**Discussion by squallsama** [Albeoris/Memoria#274] [Q&A] [answered]
Any information about what version of SDK I should use with this project and what steps I should perform to build ? Current readme is so vague...

**Comment by snouz** [comment, marked as answer]
Hi, I managed yesterday to build on this project again from a new computer:
I'm currently adding the info to the README.
What I had to do: download Visual studio 2019 community, open the project file in the repo, it gave me 4 prompts to download the specific target of .NET 3.5 or change to 4.6.1, but the link the nuget has for downloading that specific target is dead, so I had to let it change my 4 repos to 4.6.1, then for each, open them and double click properties
![image](https://github.com/Albeoris/Memoria/assets/9964576/912c4886-0cc0-4fb1-9585-8ca19ee95ca5)
and change the profile back to 3.5.
![image](https://github.com/Albeoris/Memoria/assets/9964576/04a44c3d-b4b8-474a-8f88-c46c564a299d)
Then I'm able to compile with "regenerate the solution" without error
![image](https://github.com/Albeoris/Memoria/assets/9964576/63adcb14-875f-4e4f-b9b0-8165a4198faa)

Of course make sure you don't push these change to the repo. I use github desktop, and I have to let these 4 changes unchecked.
![image](https://github.com/Albeoris/Memoria/assets/9964576/77f0b5d0-9480-420a-94b9-3ffc8bd59118)

Can some people confirm that it works for them too?

**Reply by squallsama** [reply]
Thanks ! Works for me! Also did following - download 3.5 .net from https://dotnet.microsoft.com/en-us/download/visual-studio-sdks?cid=getdotnetsdk

**Reply by snouz** [reply]
Ah yes, you have to have 3.5, but launching that did nothing for me because it's pre-installed on Win11.
